### DISCUSSION TITLE
Using Surefire (JUnix XML) reports for nicer test outputs

### DISCUSSION THREAD

**Discussion by hauleth** [community/community#25401] [Actions]
<p>It would be handy to have such support in GitHub Actions. This is one of the features I miss from the GitLab CI and CircleCI when working with GitHub Actions.</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/hauleth">@hauleth</a>,<br>
Thanks for your feedback.<br>
GitHub takes your suggestions very seriously, and the suggestions are very helpful for improving GitHub.</p>
<p>I recommend you directly share your suggestion <a href="https://support.github.com/contact/feedback?contact%5Bcategory%5D=actions" rel="nofollow noopener">here</a>. That will allow you to directly interact with the appropriate engineering team, and make it more convenient for the engineering team to collect and categorize your suggestions.</p>
